### PR TITLE
Tic tac toe api

### PR DESCRIPTION
**Pull request by DavidRojas1612**
## Summary
-Completed use tic-tac-toe-api for list matches in the tic-tac-toe game.

## Screenshots
use_api

## Trello Card
https://trello.com/c/n53lKmgu

### CODE REVIEW

File: react/tic-tac-toe/src/app/screens/Game/components/Matches/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@import 'src/scss/colors.scss';
+$square-border: $dust-gray;
+$text-player: $black;
```
Comment (alejobh) [inline react/tic-tac-toe/src/app/screens/Game/components/Matches/styles.module.scss]: Are you actually using this?

File: react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  };
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.getMatches();
+    }, 3000);
+  }
+
+  // recomendacioón de alejo
+  parseResponse = response => {
```
Comment (alejobh) [inline react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js]: This is a way to standard all your api request, but its better if you let it outside this component, in a utils file

File: react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      });
+    } catch (error) {
+      console.error('error: ', error);
+    }
+  };
+
+  render() {
+    const { matches, loading } = this.state;
+    return (
+      <Fragment>
```
Comment (alejobh) [inline react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js]: use `<>`

File: react/tic-tac-toe/src/app/screens/Game/index.js [hunk 4]
```diff
@@ -56,6 +57,10 @@ class Game extends Component {
           <div>{status}</div>
           <ol>{history.map(this.renderMoves)}</ol>
         </div>
+        <div className={styles.gameInfo}>
+          <div>Matches</div>
```
Comment (alejobh) [inline react/tic-tac-toe/src/app/screens/Game/index.js]: `Partidas`

File: react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js (hunk 2 above)
Comment (jmtov) [reply react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js]: 👆

File: react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+import styles from './styles.module.scss';
+
+class Matches extends Component {
+  state = {
+    matches: [],
+    loading: true
+  };
+
+  componentDidMount() {
+    setTimeout(() => {
```
Comment (jmtov) [inline react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js]: Why this timeout?

Comment (DavidRojas1612) [reply react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js]: this timeout simulates a slow response from the api. 😅😅

File: react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js (hunk 2 above)
Comment (DavidRojas1612) [reply react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js]: it's okay, the changes of this method have been made in the next branches. 😄🤟🏽

**Pull request merged**